request_test.go

```diff
@@ -0,0 +1,108 @@
+package profitbricks
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path/filepath"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+const requestTestDataPath = "testdata"
+
+type SuiteRequest struct {
+	suite.Suite
+	client *Client
+	apiUrl string
+}
+
+func (s *SuiteRequest) SetupTest() {
+	s.client = NewClient("", "")
+	s.apiUrl = s.client.client.apiURL
+	httpmock.Activate()
+}
+
+func (s *SuiteRequest) TearDownTest() {
+	httpmock.DeactivateAndReset()
+}
+
+type SuiteWaitTillRequests struct {
+	SuiteRequest
+}
+
+func Test_WaitTillRequests(t *testing.T) {
+	suite.Run(t, new(SuiteWaitTillRequests))
+}
+
+func loadTestData(t *testing.T, filename string) []byte {
+	data, err := ioutil.ReadFile(filepath.Join(requestTestDataPath, filename))
+	assert.NoError(t, err)
```

Comment:
just an idea. this could be a suite's method. you wouldn't need the separate import of `assert`. do as you like.

Reply:
I'd rather like to keep it this way for now. Its more likely that this becomes a shared method for all tests than copying this method for all suites. If there will be a single super suite, then ok, but lets decide this later.